project/src/main/puzzle/puzzle-tile-map.gd

```diff
@@ -26,6 +26,9 @@ const COL_COUNT = 9
 # a number in the range [0, 1] which can be set to make the tile map flash or blink.
 var whiteness := 0.0 setget set_whiteness
 
+# Pieces have a second shadow at the destination
```

Comment:
Mention "ghost" somewhere in the comment, and maybe change the field name as well to something like "ghost shadow offset"

Reply:
Changed documentation to `offset used for drawing the 'ghost piece'`, and changed variable name to include `ghost`